### PR TITLE
Comsol bessemer

### PR DESCRIPTION
**Pull request by desmfryan**
Comsol install on Bessemer documentation. 

The suggested batch script was tested successfully, producing correct output.

### CODE REVIEW

File: bessemer/software/apps/comsol.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+   :Support Level: Install Only
+   :Dependencies: None
+   :URL: https://www.comsol.com/comsol-multiphysics
+
+COMSOL Multiphysics provides a range of simulation, finite element analysis and solver functionality. 
+
+Availability, licensing and modules
+-----------------------------------
+
+COMSOL Multiphysics is installed on Bessemer but is not presently supported by CiCS.  There is no site license for COMSOL so to run it on Bessemer you will need either your own license or access to licenses managed by others.  For example, a research group may operate a *license server* that can issue COMSOL licenses to (approved) users.  If you need access to a license on a temporary basis then you could try contacting `Prof. Will Zimmerman <https://www.sheffield.ac.uk/cbe/academic-staff/professor-will-zimmerman>`_, who has purchased several licenses for his research group and may be willing to hire out licenses to other researchers.  See `Interactive usage` for information on how to specify a COMSOL license when starting COMSOL on Iceberg.
```
Comment (willfurnass) [inline bessemer/software/apps/comsol.rst]: Link to 'interactive usage' broken.

Comment (willfurnass) [reply bessemer/software/apps/comsol.rst]: Mentions Iceberg

File: bessemer/software/apps/comsol.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+After connecting to Bessemer (see :ref:`ssh`),  start an interactive graphical session with the :code:`srun --mem=2G --pty bash -i` command. 
+Alternatively, if you require more memory, for example 16 GB, use the command :code:`srun --mem=16G --pty bash -i` 
+
+Next, run the following to make COMSOL Multiphysics available in your current session: ::
+
+        module load COMSOL/5.4
+
+Specify the location of your license information: ::
+
+        export LM_LICENSE_SERVER=/home/myusername/path/to/mylicensefile.dat:$LM_LICENSE_SERVER
```
Comment (willfurnass) [inline bessemer/software/apps/comsol.rst]: Odd that LM_LICENSE_SERVER not set to /usr/local/packages/common/licenses/comsol.dat by the COMSOL EasyBlock...

File: bessemer/software/apps/comsol.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+*    LiveLink™ for Revit®
+*    LiveLink™ for PTC® Creo® Parametric™
+*    LiveLink™ for PTC® Pro/ENGINEER®
+*    LiveLink™ for Solid Edge®
+*    File Import for CATIA® V5
+
+Interactive usage
+-----------------
+
+After connecting to Bessemer (see :ref:`ssh`),  start an interactive graphical session with the :code:`srun --mem=2G --pty bash -i` command.
```
Comment (willfurnass) [inline bessemer/software/apps/comsol.rst]: Point users at https://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/submit.html#interactive-sessions for info on starting interactive SLURM job instead?

File: bessemer/software/apps/comsol.rst [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Next, run the following to make COMSOL Multiphysics available in your current session: ::
+
+        module load COMSOL/5.4
+
+Specify the location of your license information: ::
+
+        export LM_LICENSE_SERVER=/home/myusername/path/to/mylicensefile.dat:$LM_LICENSE_SERVER
+
+Finally, start COMSOL Multiphysics: ::
```
Comment (willfurnass) [inline bessemer/software/apps/comsol.rst]: Move these two lines to just above 'Serial batch usage' i.e. after all info re licensing?

File: bessemer/software/apps/comsol.rst (hunk 1 above)
Comment (desmfryan) [reply bessemer/software/apps/comsol.rst]: > 
> 
> Link to 'interactive usage' broken.

Not a link but reference to section below...will insert (below)

Comment (desmfryan) [reply bessemer/software/apps/comsol.rst]: > 
> 
> Mentions Iceberg

Oops...thought I got rid of them all

File: bessemer/software/apps/comsol.rst (hunk 2 above)
Comment (desmfryan) [reply bessemer/software/apps/comsol.rst]: The default license location is in the install license folder, so LM_LICENSE_SERVER is only set to point to a non-standard location i.e. a user having their own license file.

File: bessemer/software/apps/comsol.rst (hunk 3 above)
Comment (desmfryan) [reply bessemer/software/apps/comsol.rst]: Agree

File: bessemer/software/apps/comsol.rst (hunk 4 above)
Comment (desmfryan) [reply bessemer/software/apps/comsol.rst]: Agree

**Pull request merged**